actors/stage/src/main/java/com/ea/orbit/actors/Stage.java

```diff
@@ -77,6 +79,9 @@
     @Wired
     Container container;
 
+    @Inject
```

Comment:
Hmm, If we aren't running the container the @Config line would break as well wouldn't it since we rely on the Container to inject the config?